.eslintrc.json

```diff
@@ -1,10 +1,11 @@
 {
   "parserOptions": {
-    "ecmaVersion": 5
+    "ecmaVersion": 6,
+    "sourceType": "module"
   },
   "extends": "eslint:recommended",
   "env": {
-    "commonjs": true
+    "es6": true
```

Comment:
I don't recall if we ended up agreeing on ES6 in the eslint configurations vs. just updating the `parserOptions.sourceType` field. Do you happen to have a link to that discussion?

If we are going forward with ES6, then I'm curious if we need to set both `env: es6` and `parserOptions.ecmaVersion: 6`. What happens if we just remove `env` but still set the parser option to ecmaVersion 6?